kotlin/perform-spatial-operations/README.md

```diff
@@ -2,7 +2,7 @@
 
 Find the union, intersection, or difference of two geometries.
 
-![](perform-spatial-operations.png)
+![Image of perform spatial operations](perform-spatial-operations.png)
 
 ## Use case
 
```

Comment:
Can't comment on the stuff below, but the java comments regarding How to use this sample apply here too.